lib/internal/webstreams/readablestream.js

```diff
@@ -140,6 +141,33 @@ const kError = Symbol('kError');
 const kPull = Symbol('kPull');
 const kRelease = Symbol('kRelease');
 
+let releasedError;
+let releasingError;
+
+function lazyReadableReleasedError() {
+  if (releasedError) {
+    return releasedError;
+  }
+  releasedError = new ERR_INVALID_STATE.TypeError('Reader released');
+  // Avoid V8 leak and remove userland stackstrace
+  releasedError.stack = SideEffectFreeRegExpPrototypeSymbolReplace(
+    /^.*\((?!node:|internal)[^()]*\).*$/gm,
```

Comment:
This is a good start but it won't match all internal frames. There are e.g., not always brackets involved.
The best one to match I could come up with was: `const coreModuleRegExp = /^ {4}at (?:[^/\\(]+ \(|)node:(.+):\d+:\d+\)?$/;`

Reply:
Thanks! I've changed it a bit to perform the inverse operation (userland modules). Can you double-check?